### ISSUE TITLE
Option to embed attachment on article

### ISSUE THREAD

**Issue by theonlyringo** [thorsten/phpMyFAQ#1080]
I have ran into this issue a few times from a couple of contributors where they want the attachment to be embedded into the article. Say if the attachment was a PDF, a check box or option could be selected to display the PDF in the article as a normal object. ex. <object title="" data="full-path-to-attachment" type="application/pdf" width="100%">

**Comment by thorsten** [comment]
How should the embedded attachment look like?

**Comment by theonlyringo** [comment]
I am displaying PDFs and DOCs by just using
 ```
<object title="" data="https://fullpathto.pdf#" type="application/pdf" width="100%" height="800px">
<p>It appears your Web browser is not configured to display PDF files. No worries, just <a href="https://fullpathto.pdf">click here to download the PDF file.</a></p>
</object>
```
![ss](https://cloud.githubusercontent.com/assets/5071472/12989583/bcf41e48-d0cb-11e5-9b06-4d3aa40a5b6a.jpg)

I had been using Google docs to display the documents needed, but it seemed to only work sometimes and wasn't reliable. 

`<iframe src="http://docs.google.com/gview?url=http://fullpathto.pdf&embedded=true" style="width:600px; height:500px;" frameborder="0"></iframe>`

**Comment by thorsten** [comment]
Should be possible to add <object> via https://www.tinymce.com/docs/configure/content-filtering/#extended_valid_elements

**Comment by thorsten** [comment]
<object> support will be included in 2.8.26. Thanks!
